products/models.py

```diff
@@ -102,3 +103,15 @@ class Meta:
 
     def __str__(self):
         return f'{self.banner_name}'
+
+
+class Cart(models.Model):
+    """this model is useful to add products into cart """
+
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    product = models.ForeignKey(Product, on_delete=models.CASCADE)
```

Comment:
It's recommended to add a date and time field for auto-discarding functionality if we need to incorporate it.

Reply:
done